index.md

```diff
@@ -181,9 +181,3 @@ navigation:
     </ul>
   </div>
 </div>
-
-<div class="row">
```

Comment:
This is a redundant download link at the bottom; not strictly related, but, I removed it as it's already in the right nav.